docs/getting-involved-in-a-track.md

```diff
@@ -0,0 +1,100 @@
+# Getting Involved in an Exercism Language Track
+
+[repositories]: http://exercism.io/repositories
+[triaging-issues]: /docs/triaging-issues-in-a-track.md
+[reviewing-prs]: /docs/reviewing-a-pull-request.md
+[porting]: /docs/porting-an-exercise.md
+[xcommon]: http://github.com/exercism/x-common
+[blazon]: /docs/improving-consistency-across-tracks.md
+
+The Exercism language tracks are a great way to get involved in:
+
+- a programming language you love
+- a programming language you're curious about
+- open source (in general)
+
+Each language track lives in its own repository, which means that you can contribute
+to a track without having to know anything about the rest of the Exercism ecosystem.
+
+Also, each track is focused, containing implementations of trivial exercises and the tools
+to make development easier, so there's no big codebase to get acquainted with.
+
+## Submit a Couple of Solutions
+
+If you haven't used Exercism before, then we recommend submitting solutions to a couple of
+exercises. It doesn't matter which language track you submit to, it's just to get a feel
+for what a language track consists of.
+
+## Watch The Track Repository
+
+If you haven't picked a language you want to contribute to yet, check out the list of [language tracks
+and their respective repositories][repositories].
+
+Then go to the repository for the language you've chosen, click the _Watch_ button, and select _Watching_.
+This will notify you of any new issues, pull requests, or comments in the repository, which is a great way
+of getting acquainted with the people involved and the issues that tend to come up.
+
+## Orient Yourself
+
+Read the README, and look through the open issues and pull requests, and get a feel for what's going on.
+
+## Contribute!
+
+There are a number of ways to contribute to a track. All of them are sorely needed, and greatly appreciated!
+
+### Triaging Issues
+
+A great issue is detailed and actionable. When they're not, you can help ask the questions to make them so.
+
+For more detailed suggestions about things to keep in mind when triaging, check out [this documentation][triaging-issues].
+
+### Reviewing Pull Requests
+
+We always need more eyeballs on pull requests. On language tracks most pull requests tend to be related to
+the exercises themselves, and we have [detailed documentation][reviewing-prs] that should help
+you get started with code reviews.
+
+### Porting an Exercise
+
+The easiest way to add a new exercise is to find an exercise that has already been implemented in another language
+track, and port it over to your target language.
```

Comment:
I would be curious to see a list of the available exercises. Is https://github.com/exercism/x-common/blob/master/exercises/ the best place to start?